AZ3166/jenkins/deployment/DevKitDeploy.ps1

```diff
@@ -39,6 +39,16 @@ foreach($StorageAccountName in $StorageHashTable.Keys)
 	$ArduinoPackageBlobName = "AZ3166-" + $CurrentVersion + ".zip"
 	Set-AzureStorageBlobContent -Context $StorageContext -Container $Environment -File $ArduinoPackageFilePath -Blob "$ArduinoPackageContainer\$ArduinoPackageBlobName" -Force
 
+	# Upload Firmware bin file
+	$FirmwareFileName = "devkit-firmware-" + $CurrentVersion + "." + $env:BUILD_NUMBER + ".bin"
+	$FirmwareFilePath = Join-Path -Path (Get-Location).Path -ChildPath "TestResult\$FirmwareFileName"
+	Set-AzureStorageBlobContent -Context $StorageContext -Container $Environment -File $FirmwareFilePath -Blob $FirmwareFileName -Force
```

Comment:
confused, currently the script doesn't upgrade the firmware to blob automatically ?

Reply:
We have removed it before in https://github.com/Microsoft/devkit-sdk/pull/855 .